dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/CustomAccessLogFieldsTest.java

```diff
@@ -288,7 +288,7 @@ public void testAllFieldsArePrinted() throws Exception {
         isFirstTimeUsingXmlServerWithMetrics = false;
 
         hitHttpsEndpointSecure("/metrics", xmlServerWithMetrics);
-        String line = xmlServerWithMetrics.waitForStringInLog("liberty_accesslog");
+        String line = xmlServerWithMetrics.waitForStringInLog("liberty_accesslog.*/metrics");
```

Comment:
There are other occurrences of `waitForStringInLog("liberty_accesslog")`.  Do you need to change them all?

Reply:
They did not appear in the test failure defect but I changed all the relevant ones (i.e. the ones that hit /metrics endpoint) just in case.